### ISSUE TITLE
Training and Validation Accuracy (with overfitting)

### ISSUE THREAD

**Issue by AhmadSabbirChowdhury** [AhmadSabbirChowdhury/Cats-Vs-Dogs-Classification-Kaggle-Problem-Set-#1]
![Inaccurate Data](https://user-images.githubusercontent.com/45898995/82493935-9b8e6480-9b0a-11ea-952f-d5425fe17561.png)
